package.json

```diff
@@ -83,5 +202,8 @@
   },
   "devDependencies": {
     "grunt": "~0.4.5"
+  },
+  "scripts": {
+    "prepublish": "node utils/updatePackage.js"
```

Comment:
Ah, ok. I didn't know about that phase. I'm also a fan of removing this for now and making it manual.